core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -266,7 +266,10 @@ class CivilizationInfo {
     //region pure functions
     fun getDifficulty(): Difficulty {
         if (isPlayerCivilization()) return gameInfo.getDifficulty()
-        return gameInfo.ruleSet.difficulties["Chieftain"]!!
+        // TODO We should be able to mark a difficulty as 'default AI difficulty' somehow
```

Comment:
ModConstants maybe?

Reply:
Perhaps, that's for a different PR though :)